docs/src/main/asciidoc/spanner.adoc

```diff
@@ -1343,6 +1343,26 @@ class SpringSecurityAuditorAware implements AuditorAware<User> {
 
 You can also set a custom provider for properties annotated `@LastModifiedDate` by providing a bean for `DateTimeProvider` and providing the bean name to `@EnableSpannerAuditing(dateTimeProviderRef = "customDateTimeProviderBean")`.
 
+=== Multi-Instance Usage
+
+Your application can be configured to use multiple Cloud Spanner instances or databases by providing a custom bean for `DatabaseIdProvider`.
+The default bean uses the instance ID, database name, and project ID options you configured in `application.properties`.
+
+[source,java]
+----
+    @Bean
+    public DatabaseIdProvider databaseIdProvider() {
+        // return custom connection options provider
+    }
+----
+
+The `DatabaseId` given by this provider is used as the target database name and instance of each operation Spring Data Cloud Spanner executes.
+By providing a custom implementation of this bean (for example, supplying a thread-local `DatabaseId`), you can direct your application to use multiple instances or databases because.
```

Comment:
grammar error - delete "because" or add more to sentence?

Reply:
LOL oh that's a typo.